src/main/java/com/internet/shop/models/Product.java

```diff
@@ -0,0 +1,47 @@
+package com.internet.shop.models;
+
+public class Product {
+    private Long id;
+    private String name;
+    private Double price;
```

Comment:
```suggestion
    private double price;
```
using a wrapper is recommended for id, but for this field we can use a primitive type

Reply:
Done